history/views.py

```diff
@@ -564,7 +567,8 @@ def profit_view(request):
     # get data
     data = {}
     for t in Trade.objects.filter(symbol=symbol, status='fill').order_by('-created_on').all():
-        date = datetime.datetime.strftime(t.created_on-datetime.timedelta(hours=7), '%Y-%m-%d')
+        mst_dt = timezone.localtime(t.created_on, pytz.timezone('MST'))
+        date = datetime.datetime.strftime(mst_dt, '%Y-%m-%d')
```

Comment:
> could be solved with a migration

Well in that case we would have to store the data in UTC, but at that point it would be useless. We could remove `created_on_str` and add an attribute on the fly. Or use a Python property. Is it possible? Currently I don't have data to create a chart.